mopidy_mpris/objects.py

```diff
@@ -49,7 +49,15 @@ def _get_root_iface_properties(self):
             'SupportedUriSchemes': (self.get_SupportedUriSchemes, None),
             # NOTE Return MIME types supported by local backend if support for
             # reporting supported MIME types is added
-            'SupportedMimeTypes': (dbus.Array([], signature='s'), None),
+            'SupportedMimeTypes': (dbus.Array([
+                dbus.String(u'audio/mpeg'),
```

Comment:
The `u` string prefixes here are redundant, as the file starts with `from __future__ import unicode_literals`.